### PR TITLE
add new term MONDO:0100138 X-linked recessive mitochondrial myopathy

### LINKED ISSUE #1585: [NTR] X-linked recessive mitochondrial myopathy
Maybe this disease already exists  under a different name and this is just a new gene association? 

An  X-linked recessive mitochondrial myopathy caused by defects in the MICOS subunit gene APOO (MIC26). Modelling in yeast and flies demonstrate an inability to insert MICOS complex into the inner mitohondrial membrane. Associated symptoms include, lactic acidosis, cognitive impairment and autistic features. 
PMID: 32439808

parents is_a
neuromuscular disease (MONDO:0019056)
mitochondrial disease (MONDO:0044970)
X-linked recessive disease (MONDO:0020605)
others?

### PR DESCRIPTION
**Pull request by nicolevasilevsky**
closes #1585

### CODE REVIEW

File: src/ontology/mondo-edit.obo [hunk 1]
```diff
@@ -433092,6 +433092,19 @@ synonym: "PARDS" EXACT [PMID:25647235, PMID:31236307]
 is_a: MONDO:0006502 ! acute respiratory distress syndrome
 created_by: http://orcid.org/0000-0001-5208-3432
 
+[Term]
+id: MONDO:0100138
+name: X-linked recessive mitochondrial myopathy
+def: "A mitochondrial myopathy caused by defects in the MICOS subunit gene APOO (MIC26). Modelling in yeast and flies demonstrate an inability to insert MICOS complex into the inner mitohondrial membrane. Associated symptoms include, lactic acidosis, cognitive impairment and autistic features." [PMID:32439808]
```
Comment (cmungall) [inline src/ontology/mondo-edit.obo:433098]: mild concern with deviation between ldef and tdef but we see this a lot with MoI defined diseases

Comment (nicolevasilevsky) [reply src/ontology/mondo-edit.obo:433098]: I'm not sure what you mean here. Is any action needed?

**Comment by nicolevasilevsky** [comment]
- [ ] fix conflict and okay to merge

**Comment by nicolevasilevsky** [comment]
the check passed

**Pull request merged**